XML/TestCases/FunctionalTests-GPU.xml

```diff
@@ -27,43 +27,30 @@
         <Priority>0</Priority>
     </test>
     <test>
-        <testName>NVIDIA-CUDA-DRIVER-TENSORFLOW</testName>
-        <preTestScript>GPU-DRIVER-INSTALL.ps1</preTestScript>
-        <testScript>GPU-TENSORFLOW.ps1</testScript>
+        <testName>NVIDIA-CUDA-DRIVER-VALIDATION-MAX-GPU</testName>
+        <testScript>GPU-DRIVER-INSTALL.ps1</testScript>
         <setupType>OneVM</setupType>
-        <OverrideVMSize>Standard_NC6</OverrideVMSize>
-        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\gpu-driver-install.sh,.\Testscripts\Linux\gpu-tensorflow.sh</files>
+        <OverrideVMSize>Standard_NC24</OverrideVMSize>
+        <files>.\Testscripts\Linux\gpu-driver-install.sh,.\Testscripts\Linux\utils.sh</files>
         <TestParameters>
             <param>CUDADriverVersion="CUDA_DRIVER"</param>
-            <param>CudaToolkitVersion="CUDATOOLKIT_VERSION"</param>
```

Comment:
why are they removed?

Reply:
> why are they removed?

This is github showing the diff incorrectly as I moved tensorflow tests and added the MAX ones.
Please check the full file with the new changes, the params are in good order.